packages/plugin-ext/src/plugin/document-data.ts

```diff
@@ -127,7 +128,7 @@ export class DocumentDataExt {
 
         // Insert new lines & store lengths
         const newLengths = new Uint32Array(insertLines.length - 1);
-        for (let i = 1; i < insertLines.length; i++) {
+        for (let i = 1;i < insertLines.length;i++) {
```

Comment:
I believe that when CI runs, this will cause a lint error. Please restore the original spacing, here and elsewhere in this file.

Reply:
Sorry, it was my negligence. I has reverted the code here. Thank you.